composer.json

```
@@ -16,12 +16,17 @@
         "php": ">=5.6"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8",
+        "phpunit/phpunit": "^5.7 || ^6.5",
```

Since we're changing the test cases to extend `PHPUnit\Framework\TestCase` v5.X will no longer work. Instead we can do:

```suggestion
        "phpunit/phpunit": "^6.0 || ^7.0",
```

---

You're right. Keep defining the latest/current PHPUnit versions is correct.